packages/remix-dev/config.ts

```diff
@@ -401,6 +412,8 @@ export async function readConfig(
 
   let serverDependenciesToBundle = appConfig.serverDependenciesToBundle || [];
 
+  let warnOnFakeBuiltins = appConfig.warnOnFakeBuiltins? appConfig.warnOnFakeBuiltins: false
```

Comment:
This can be simplified to

```suggestion
  let warnOnFakeBuiltins = appConfig.warnOnFakeBuiltins ?? false;
```